### PR TITLE
qmp_command:add command "query-target"

### PR DESCRIPTION
**Pull request by yiqianwei**
qmp_command:add command "query-target"

ID:1823311

Signed-off-by: Yiqian Wei <yiwei@redhat.com>

### CODE REVIEW

**Comment by yiqianwei** [comment]
@jingzhao84 @yanan-fu  Could you review this patch?

**Comment by yiqianwei** [comment]
On x86 host:
 (1/1) Host_RHEL.m8.u2.product_av.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.2.0.x86_64.io-github-autotest-qemu.qmp_command.qmp_query-target: PASS (49.11 s)
RESULTS    : PASS 1 | ERROR 0 | FAIL 0 | SKIP 0 | WARN 0 | INTERRUPT 0 | CANCEL 0
JOB TIME   : 51.64 s
JOB HTML   : /root/avocado/job-results/job-2020-04-14T23.36-ce34c7a/results.html

On Power host:
  (1/1) Host_RHEL.m8.u2.product_rhel.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.2.0.ppc64le.io-github-autotest-qemu.qmp_command.qmp_query-target: PASS (103.37 s)
RESULTS    : PASS 1 | ERROR 0 | FAIL 0 | SKIP 0 | WARN 0 | INTERRUPT 0 | CANCEL 0
JOB TIME   : 111.29 s
JOB HTML   : /root/avocado/job-results/job-2020-04-14T23.40-390a02c/results.html

**Comment by xutian** [comment]
Looks OK to me.

File: qemu/tests/cfg/qmp_command.cfg [hunk 1]
```diff
@@ -183,3 +183,7 @@
             qmp_cmd = "query-command-line-options"
             cmd_result_check = contain
             cmd_return_value = "[{'option': 'option-rom'}, {'name': 'bootindex'}, {'name': 'romfile'}]"
+        - qmp_query-target:
+            qmp_cmd = "query-target"
+            cmd_result_check = equal
+
```
Comment (yanan-fu) [inline qemu/tests/cfg/qmp_command.cfg:189]: Please remove this blank line at EOF.

File: qemu/tests/qmp_command.py [hunk 2]
```diff
@@ -245,6 +245,12 @@ def send_cmd(cmd):
                 image_info['qdev'] = image
                 image_info['format'] = image_format
                 expect_o.append(image_info)
+        elif qmp_cmd == "query-target":
+            host_arch = platform.machine()
+            if host_arch == "ppc64le":
+                host_arch = host_arch[0:5]
+                expect_o = [{"arch": host_arch}]
+            expect_o = [{"arch": host_arch}]
```
Comment (yanan-fu) [inline qemu/tests/qmp_command.py:253]:
```
        elif qmp_cmd == "query-target":
            host_arch = platform.machine()
            if host_arch == "ppc64le":
                host_arch = host_arch[:5]
            expect_o = [{"arch": host_arch}]
```

File: qemu/tests/cfg/qmp_command.cfg (hunk 1 above)
Comment (yiqianwei) [reply qemu/tests/cfg/qmp_command.cfg]: Okay,this blank line has been deleted.

File: qemu/tests/qmp_command.py (hunk 2 above)
Comment (yiqianwei) [reply qemu/tests/qmp_command.py:252]: i got it,thanks

**Comment by yanan-fu** [comment]
Ack

**Pull request merged**